_posts/2017-03-28-table-sql-api-update.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: post
+title:  "From Streams to Tables and Back Again: An Update on Flink's Table & SQL API"
+excerpt: "<p>Broadening the user base and unifying batch & streaming with relational APIs</p>"
+date:   2017-03-27 12:00:00
+author: "Timo Walther"
+author-twitter: "twalthr"
+categories: news
+---
+Here’s the thing about stream processing: it offers a lot of value, but it sure can be hard. Many organizations have recognized the benefit of managing large volumes of data in real-time, reacting quickly to trends, and providing customers with live services at scale. Streaming applications with well-defined business logic can deliver a competitive advantage.
+
+But streaming concepts such as windows, event time, out-of-order event handling, and state management are challenging for many users who could benefit from streaming. Most streaming APIs require deep Java and Scala knowledge, especially when it comes to efficient programming, thus limiting the range of users who can interact with modern streaming frameworks.
```

Comment:
@twalthr I think that replacing the second paragraph with your rewrite would work well